### PR TITLE
[fix] package configs in sdist package

### PR DESCRIPTION
**Pull request by ariwaranosai**
add config in MANIFEST.in to package yaml in mmf/configs.

see #479

### CODE REVIEW

**Comment by vedanuj** [comment]
Thanks for the fix. Looks good to me. Will leave it to @apsdehal  to approve.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,3 +1,4 @@
 include requirements.txt
 include LICENSE
 include NOTICES
+recursive-include mmf/configs/ *.yaml
```
Comment (apsdehal) [inline MANIFEST.in:4]: Can you also `recursive-include mmf/projects/ *.yaml`?

Comment (ariwaranosai) [reply MANIFEST.in:4]: right,  fixed.

**Comment by apsdehal** [comment]
Thanks for the PR. This has now landed into the master.

**Pull request closed without merging**